modules/core/src/main/scala/gem/enum/package.scala

```diff
@@ -1,21 +1,10 @@
 package gem
 
-import gem.config._
-
 // The members of this package are generated from database tables, which are the source of truth.
 // See project/gen2.scala for details. Associations with other model types, as needed, are provided
 // here as implicit classes wrapping the generated companion objects.
 package object enum {
 
-  /** Add mapping from InstrumentConfig to Instrument. */
-  implicit class InstrumentCompanionOps(companion: Instrument.type) {
```

Comment:
We get this now by asking for `instrument` directly on the static or dynamic config?